Documentation/Settings.cfg

```diff
@@ -1,21 +1,62 @@
-[general]
-
-project = Surf
-release = master
-version = 3.3
+# More information about this file:
+# https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#settings-cfg
 
-[notify]
+[general]
 
-about_new_build = no
+project     = Surf
+version     = main
+release     = main
+copyright   = since 2011 by the contributors
 
 [html_theme_options]
 
-github_branch             = master
-github_repository         = TYPO3/Surf
-path_to_documentation_dir = Documentation
+# "Edit on GitHub" button
+github_repository    = TYPO3/Surf
+github_branch        = master
 
-project_home       = https://github.com/TYPO3/Surf
-project_issues     = https://github.com/TYPO3/Surf/issues
-project_repository = https://github.com/TYPO3/Surf
+# Footer links
+project_home         = https://docs.typo3.org/other/typo3/surf/main/en-us/
+project_contact      = https://typo3.slack.com/archives/C02EG058U
+project_repository   = https://github.com/TYPO3/Surf
+project_issues       = https://github.com/TYPO3/Surf/issues
+project_discussions  =
+
+use_opensearch       =
 
 [intersphinx_mapping]
+
+# Official TYPO3 manuals
+# h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/
+# t3cheatsheets  = https://docs.typo3.org/m/typo3/docs-cheatsheets/main/en-us/
+# t3contribute   = https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/
+# t3coreapi      = https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/
+# t3docteam      = https://docs.typo3.org/m/typo3/team-t3docteam/main/en-us/
+# t3editors      = https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/
+# t3extbasebook  = https://docs.typo3.org/m/typo3/book-extbasefluid/main/en-us/
+# t3extexample   = https://docs.typo3.org/m/typo3/guide-example-extension-manual/main/en-us/
+# t3home         = https://docs.typo3.org/
+# t3install      = https://docs.typo3.org/m/typo3/guide-installation/main/en-us/
```

Comment:
they can all be dropped since this is not a typo3 thing

Reply:
i would leave them in, as they give guidance for possible cross references, but of course you can remove them.